index.bs

```diff
@@ -1008,10 +1007,13 @@ Individual Participation Criteria</h3>
 	“Disclosure”
 	in the W3C Patent Policy [[!PATENT-POLICY]].
 
-	The [=Director=] <em class="rfc2119">may</em> suspend or remove for cause
-	a participant in any group (including the [=AB=] and [=TAG=]),
-	where cause includes failure to meet the requirements of this process,
-	the membership agreement, or applicable laws.
+	When serious and/or repeated violations do occur,
+	including failure to meet the requirements of this process,
+	the membership agreement, or applicable laws,
+	and repeated attempts to address these violations have not resolved the situation,
```

Comment:
The allowance of repeated attempts to address violations applies to the processes. It is not necessary for all violations, and indeed should not be required for e.g. egregious violations of the law or people's physical safety. r- without adjusting this merge of two paragraphs to maintain their previous meaning.

Reply:
Good point. How about:

> When serious and/or repeated violations do occur,
> including failure to meet the requirements of this process 
> where repeated attempts to address these violations have not resolved the situation,
> or violation of the membership agreement or applicable laws,
> the [=Director=] <em class="rfc2119">may</em> take disciplinary action,
> including suspending or removing for cause
> a participant in any group (including the [=AB=] and [=TAG=]).

I think this scope it better, and to be equivalent to the original phrasing. I do wonder if egregious violations of the CEPC (which fall under "failure to meet the requirements of this process") should require *repeated* attempts to resolve the situation, or if exclusion can be more immediate, but maybe this should be open as a separate issue, to keep this one editorial.